packages/e2e/features/ui/components/storage/storage-manager/upload-file-to-S3-public-multipart.feature

```diff
@@ -0,0 +1,18 @@
+
+Feature: Upload a file to S3 with public access level settings
```

Comment:
Should we make the name distinct from the default one?

Reply:
Good catch!